sql/telemetry_derived/error_aggregates_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Error aggregates
```

Comment:
This would change the destination table name to `error_aggregates_v1` which would also need to be changed here: https://github.com/mozilla/telemetry-airflow/blob/master/dags/error_aggregates.py

We might also want to create a `error_aggregates` view to ensure nothing breaks.